brmo-loader/src/main/resources/xsl/brk-snapshot-to-rsgb-xml.xsl

```diff
@@ -155,27 +155,25 @@
             <xsl:variable name="vve_id" select="../recht:ZakelijkRecht[substring(recht:rustOp/KadastraalObjectRef:AppartementsrechtRef/@xlink:href,2) = $app_re_id]//recht:verenigingVanEigenaren[1]/PersoonRef:KADNietNatuurlijkPersoonRef/@xlink:href" xmlns:KadastraalObjectRef="http://www.kadaster.nl/schemas/brk-levering/snapshot/imkad-kadastraalobject-ref/v20120201" xmlns:PersoonRef="http://www.kadaster.nl/schemas/brk-levering/snapshot/imkad-persoon-ref/v20120201"/>
-            <xsl:if test="$vve_id">
-                <fk_2nnp_sc_identif>
-                    <xsl:choose>
-                        <xsl:when test="../pers:KADNietNatuurlijkPersoon[@id = substring($vve_id,2)]/pers:identificatie">
-                            <xsl:call-template name="nen_identificatie">
-                                <xsl:with-param name="id" select="../pers:KADNietNatuurlijkPersoon[@id = substring($vve_id,2)]/pers:identificatie"/>
-                            </xsl:call-template>
-                        </xsl:when>
-                        <xsl:otherwise>
-                            <xsl:call-template name="nen_identificatie">
-                                <xsl:with-param name="id" select="../nhr:Rechtspersoon[@id = substring($vve_id,2)]/pers:identificatie"/>
-                            </xsl:call-template>
-                        </xsl:otherwise>
-                    </xsl:choose>
-                </fk_2nnp_sc_identif>
-            </xsl:if>
+            <xsl:variable name="vvenhr_id" select="../recht:ZakelijkRecht[substring(recht:rustOp/KadastraalObjectRef:AppartementsrechtRef/@xlink:href,2) = $app_re_id]//recht:verenigingVanEigenaren[1]/NhrRechtspersoonRef:RechtspersoonRef/@xlink:href" xmlns:KadastraalObjectRef="http://www.kadaster.nl/schemas/brk-levering/snapshot/imkad-kadastraalobject-ref/v20120201" xmlns:NhrRechtspersoonRef="http://www.kadaster.nl/schemas/brk-levering/snapshot/imkad-nhr-rechtspersoon-ref/v20120201"/>
+            <xsl:choose>
+                <xsl:when test="$vve_id and ../pers:KADNietNatuurlijkPersoon[@id = substring($vve_id,2)]/pers:identificatie">
+                    <fk_2nnp_sc_identif>
+                        <xsl:call-template name="nen_identificatie">
+                            <xsl:with-param name="id" select="../pers:KADNietNatuurlijkPersoon[@id = substring($vve_id,2)]/pers:identificatie"/>
+                        </xsl:call-template>
+                    </fk_2nnp_sc_identif>
+                </xsl:when>
+                <xsl:when test="$vvenhr_id and ../nhr:Rechtspersoon[@id = substring($vvenhr_id,2)]/pers:identificatie">
+                    <fk_2nnp_sc_identif>
+                        <xsl:call-template name="nen_identificatie">
+                            <xsl:with-param name="id" select="../nhr:Rechtspersoon[@id = substring($vvenhr_id,2)]/pers:identificatie"/>
+                        </xsl:call-template>
+                    </fk_2nnp_sc_identif>
+                </xsl:when>
+            </xsl:choose>
```

Comment:
@cvanlith 
Deze aanpassing maakt een testcase stuk, namelijk het geval waar de verwijzing van PersoonRef:KADNietNatuurlijkPersoonRef naar een NhrRechtspersoon:Rechtspersoon is. (in dat geval is er geen `$vvenhr_id`, wel een `$vve_id` maar die verwijst naar een nhr persoon.

Ik denk dat er nog een `when` bij moet..

```xml
                <xsl:when test="$vve_id and ../nhr:Rechtspersoon[@id = substring($vvenhr_id,2)]/pers:identificatie">
                    <fk_2nnp_sc_identif>
                        <xsl:call-template name="nen_identificatie">
                            <xsl:with-param name="id" select="../nhr:Rechtspersoon[@id = substring($vve_id,2)]/pers:identificatie"/>
                        </xsl:call-template>
                    </fk_2nnp_sc_identif>
                </xsl:when>
```

Deze testcase is het eerste bericht in de Mantis6235IntegrationTest.

Is er een bericht wat eerst mis ging?

Reply:
zie https://github.com/B3Partners/brmo/pull/701